ckanext/canada/templates/internal/user/login.html

```diff
@@ -52,7 +52,7 @@
     {% endif %}
 
     {% block form %}
-      {% snippet 'user/login_form.html', action=c.login_handler, error_summary=error_summary %}
+      {% snippet 'user/snippets/login_form.html', action=c.login_handler, error_summary=error_summary %}
```

Comment:
You tell me :-) you're either fixing a bug (error_summary=errors was wrong) or you're introducing a bug. If you're fixing a bug that should be done in ckanext-wet-boew. In either case you should remove this block from ckanext-canada so it doesn't override the one in ckanext-wet-boew.